app/models/user.rb

```diff
@@ -158,45 +158,36 @@ def related_hmis_user(data_source)
 
   # list any cohort this user has some level of access to
   def cohorts
-    # START_ACL cleanup after ACL migration is complete
-    if using_acls?
```

Comment:
Don't we still need this check, since not everyone has switched over to using acls yet?

Reply:
The check is in `ids_for_relations` now, since both branches were calling `GrdaWarehouse::Cohort.where(id:` with different ids.  I may have gotten it wrong, but should be down on line 185.